MineColab.ipynb

```diff
@@ -5,25 +5,14 @@
     "colab": {
       "name": "MineColab",
       "provenance": [],
-      "collapsed_sections": [],
-      "include_colab_link": true
```

Comment:
Related to the comment below regarding the "open in Colab" button
![Screen Shot 2020-09-29 at 11 51 27 AM](https://user-images.githubusercontent.com/25097841/94520570-1d756380-024a-11eb-8c05-eecc712d1ead.png)